### PR TITLE
Adds a mint condition clear PDA to the Goodies menu for 100,000 credits.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Adds a mint condition clear PDA to the Goodies menu for 100,000 credits.

## Why It's Good For The Game

Encourages crewmembers to hustle for that sweet, sweet mint condition in box clear PDA.

## Changelog

:cl:
add: Adds a mint condition clear PDA to the Goodies menu for 100,000 credits.
/:cl:

### CODE REVIEW

**Comment by Donglesplonge** [comment]
you have doomed me and my soul to trying to trick a cargo tech into ordering this somehow its gonna be so worth it

File: code/modules/cargo/goodies.dm [hunk 1]
```diff
@@ -4,6 +4,12 @@
 	group = "Goodies"
 	goody = TRUE
 
+/datum/supply_pack/goody/clear_pda
+	name = "Mint Condition Nanotrasen Clear PDA"
+	desc = "Mint condition, freshly repackaged! A valuable collector's item normally valued at over 2.5 million credits, now available for a steal!"
```
Comment (nikothedude) [inline code/modules/cargo/goodies.dm:9]: it should come in a seamless preservative plastic box and wrap you need a knife to cut through (realism)

**Comment by EricZilla** [comment]
Add a certificate of authenticity

**Comment by Admiral-Gigaton** [comment]
make a little SecHUD icon for people who buy it, its just a dollar sign. tells them to bully this nerd and steal their PDA- i mean, to treat them with respect as they are a powerful individual

**Comment by CCC23** [comment]
time to flex on the plebians with my 1 million dollar bike and 100 grand pda

**Pull request merged**